server/apiCalls/transformers/memosCount.test.js

```diff
@@ -159,8 +160,8 @@ describe('Count memos and courses for one course offering for the same school AB
 describe('Count memos and courses as one for one course with two course offerings starting at the same time', () => {
   test('One course (two offerings) and offerings has the same memo published before course start', async () => {
     const offeringsWithMemos = [
-      offering1WithDates({ memoValues: { ladokRoundIds: ['1', '2'] }, offeringValues: { offeringId: '1' } }),
-      offering1WithDates({ memoValues: { ladokRoundIds: ['1', '2'] }, offeringValues: { offeringId: '2' } }),
+      offering1WithDates({ memoValues: { applicationCodes: ['1', '2'] }, offeringValues: { offeringId: '1' } }),
```

Comment:
`offeringId`  has to be removed to be sure tests working properly or replaced

Reply:
Removed